src/locales/en.json

```diff
@@ -296,16 +296,16 @@
       "instruction": "Make sure all the document is in the photo"
     },
     "invalid_type": {
-      "message": "File not uploading",
-      "instruction": "Try using another file type"
+      "message": "File not uploading.",
```

Comment:
is this also reflected in the spreadsheet?

Reply:
I have checked the spreadsheet, it does not include the full stop.

Reply:
The spreadsheet hasn't been updated for a long time and AFAIK is out of synch with the translations because the design team has been editing the translation directly in the document. I raised this point in the past but we haven't agreed on actions to take.

Reply:
That's my understanding of the current state of the spreadsheet also.

Reply:
cool!